contrib/apparmor/apparmor_test.go

```diff
@@ -0,0 +1,88 @@
+package apparmor
```

Comment:
Should have build tag (`// +build linux`) and "Copyright The containerd Authors." header